### PR TITLE
add updates for customer country based vat checks

### PR DESCRIPTION
**Pull request by wulfraem**
- move checks entirely into edge server
- add key press timeout for checks
- add display for validation errors on right side
- hide amount if invalid
- reduce countries in selection to eu countries
- [CORE-395]

[CORE-395]: https://evannetwork.atlassian.net/browse/CORE-395

### CODE REVIEW

File: dapps/profile/src/components/buy-eve/buy-eve.html [hunk 1]
```diff
@@ -290,13 +290,21 @@ <h3>{{ '_dappprofile.pay-eve.sepa' | translate }}</h3>
...
-          <div text-center padding-top>
-            <h5>{{ '_dappprofile.pay-eve.incl' | translate }} {{ (payEve.country == 'DE' ? '19' : '0')}} % {{ '_dappprofile.pay-eve.vatPercent' | translate }}</h5>
+          <div *ngIf="vatError" text-center padding-top>
+            <ion-chip padding class="warning-hint" *ngIf="payEve.vat && !isValidVat" color="warning">
+              <h5 no-margin>{{ ('_dappprofile.pay-eve.payment-error.' + vatError) | translate }}</h5>
+            </ion-chip>
           </div>
-          <div text-center padding-top>
+          <div *ngIf="!vatError && payEve.amount >= 10" text-center padding-top>
+            <h1>{{ ((payEve.amount || 0) * (1 + tax / 100)).toFixed(2) }}€</h1>
```
Comment (karladler) [inline dapps/profile/src/components/buy-eve/buy-eve.html]: add space before € sign ?

File: dapps/profile/src/components/buy-eve/buy-eve.ts [hunk 2]
```diff
@@ -129,7 +129,7 @@ export class EvanBuyEveComponent extends AsyncComponent implements AfterViewInit
...
+  public isoCountries = [{"Code":"AD","Name":"Andorra"},{"Code":"AT","Name":"Austria"},{"Code":"BE","Name":"Belgium"},{"Code":"BG","Name":"Bulgaria"},{"Code":"HR","Name":"Croatia"},{"Code":"CY","Name":"Cyprus"},{"Code":"CZ","Name":"Czech Republic"},{"Code":"DK","Name":"Denmark"},{"Code":"EE","Name":"Estonia"},{"Code":"FI","Name":"Finland"},{"Code":"FR","Name":"France"},{"Code":"DE","Name":"Germany"},{"Code":"EL","Name":"Greece"},{"Code":"HU","Name":"Hungary"},{"Code":"IE","Name":"Ireland"},{"Code":"IT","Name":"Italy"},{"Code":"LV","Name":"Latvia"},{"Code":"LT","Name":"Lithuania"},{"Code":"LU","Name":"Luxembourg"},{"Code":"MT","Name":"Malta"},{"Code":"NL","Name":"Netherlands"},{"Code":"PL","Name":"Poland"},{"Code":"PT","Name":"Portugal"},{"Code":"RO","Name":"Romania"},{"Code":"SK","Name":"Slovakia"},{"Code":"SI","Name":"Slovenia"},{"Code":"ES","Name":"Spain"},{"Code":"SE","Name":"Sweden"},{"Code":"GB","Name":"United Kingdom"}];
```
Comment (karladler) [inline dapps/profile/src/components/buy-eve/buy-eve.ts]: I think Andora shouldn't be in the list. UK may be removed soon ;)

**Review by karladler — APPROVED** [review]
check comments before merge

File: dapps/profile/src/components/buy-eve/buy-eve.ts (hunk 2 above)
Comment (wulfraem) [reply dapps/profile/src/components/buy-eve/buy-eve.ts]: Andorra was included in the original list, clarified it with PGR, can be removed.
UK will be dealt with, when it is time... ^^

**Pull request merged**